README.md

```diff
@@ -1 +1,325 @@
-# final-capstone-project-frontend
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] This readme seems to be duplicated with another.  It seems that it has been pasted twice.  Please, take a look at it the beginning seems like the readme for another project (like the backend).

- [ ] Your readme has instructions for tests, but there are none in your frontend project.  Kindly either add some, or remove the section.

- [ ] Kindly remove the old pieces of commented code found in `EditUser.js, CarCard.js, bookings.js, cars.js, index.js, and App.css`.  👍